### PR TITLE
displayedYogaAsanaCard

### PR DESCRIPTION
**Pull request by Nimbaland**
Ich want to see im Yoga Pose SVG which is in a Image Folder in public and import that in my db.json and then displayed on my Card component.

### CODE REVIEW

File: components/AsanaCard/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//import Image from "next/image";
+import styled from "styled-components";
+
+export default function AsanaCard({asana}) {
+  console.log(asana.image);
```
Comment (stefankorder) [inline components/AsanaCard/index.js]: Please delete this console log, inside your final PR should be no console logs.

**Review by stefankorder — APPROVED** [review]
Good work! Take a look at my comment.

File: components/AsanaCard/index.js (hunk 1 above)
Comment (Nimbaland) [reply components/AsanaCard/index.js]: Oh thank, i forgot it and change it. should i do a new PR?

File: __tests__/index.test.js [hunk 2]
```diff
@@ -1,15 +1,13 @@
-import {render, screen} from "@testing-library/react";
+import {render} from "@testing-library/react";
 import Home from "../pages/index";
 import "@testing-library/jest-dom";
 
 describe("Home", () => {
-  it("renders an octopus", () => {
+  it("renders home h1", () => {
```
Comment (stefankorder) [inline __tests__/index.test.js]: I would write: `Home component` instead of `home h1`.

File: components/AsanaCard/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+//import Image from "next/image";
```
Comment (stefankorder) [inline components/AsanaCard/index.js]: Please delete this unused code.

File: components/AsanaCard/index.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+const StyledAsanaCard = styled.article`
+  margin: 0;
+  padding: 3rem 3rem 3rem 3rem;
+  min-height: calc(100vh - 8rem);
+  margin-top: 0;
+  //background-color: lightcyan;
+`;
+
+const TheImage = styled.img`
```
Comment (stefankorder) [inline components/AsanaCard/index.js]: You better should name it `Image`

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Please take a look at my comments.

**Review by stefankorder — APPROVED** [review]
LGTM!

**Pull request merged**